lib/ansible/cli/adhoc.py

```diff
@@ -153,7 +153,7 @@ def run(self):
         elif self.options.one_line:
             cb = 'oneline'
         else:
-            cb = 'minimal'
+            cb = 'minimal' if C.DEFAULT_STDOUT_CALLBACK == 'default' else C.DEFAULT_STDOUT_CALLBACK
```

Comment:
adhoc has an extra config `bin_ansible_callbacks` which should determine this, not just stdout_callback